unity-client/Assets/Scripts/MainScripts/DCL/Environment/PlatformContext/PlatformContext.cs

```diff
@@ -0,0 +1,37 @@
+﻿using DCL.Rendering;
+
+namespace DCL
+{
+    public class PlatformContext : System.IDisposable
+    {
+        public readonly MemoryManager memoryManager;
+        public readonly ICullingController cullingController;
+        public readonly Clipboard clipboard;
+        public readonly PhysicsSyncController physicsSyncController;
+        public readonly IParcelScenesCleaner parcelScenesCleaner;
+        public readonly DebugController debugController;
```

Comment:
We use the `readonly` pattern instead of private setters to avoid the access performance penalty.